src/configs/index.js

```diff
@@ -546,3 +546,22 @@ export const ITEMS_PER_PAGE = [
     value: 30
   }
 ];
+
+export const CHECKOUT_INPUT_FIELD = {
```

Comment:
I think we can remove this consts, cause attribute equals to value all the timte.